python/example_code/dynamodb/README.md

```diff
@@ -125,6 +132,20 @@ command prompt:
 python dynamo_batching.py
 ```  
 
+### Query a table using PartiQL
+
+These two scenarios show you how to run PartiQL statements, either one at a time or in 
+batches, to query a DynamoDB table of movie data. Both scenarios add, get, update, 
+and delete movies in the table.
```

Comment:
These two scenarios show you how to run PartiQL statements to query a DynamoDB table of movie data. You can do this one at a time or in batches. Both scenarios add, get, update, and delete movies in the table.

Reply:
Fixed.